### DISCUSSION TITLE
Distorted Video Playback Issue

### DISCUSSION THREAD

**Discussion by GirardT** [aclap-dev/video-downloadhelper#300] [Help and Q&A]
### Your question

I downloaded the video with a valid license, but it appears distorted and garbled during playback. Additionally, there is no audio.

![image](https://github.com/aclap-dev/video-downloadhelper/assets/5810801/bf5e9c4d-5776-41f9-a593-3955af320af9)

### Addon & CoApp version and details

version: 8.2.1.1
channel: stable
build date: 2024-02-19
build options: linuxlic: false, noyt: true, browser: chrome
lang: en-US
license: accepted
key (only 16 first characters): 75ab156fa1284025
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/123.0.0.0 Safari/537.36
coapp: yes. Info:
{
  "id": "net.downloadhelper.coapp",
  "name": "VdhCoApp",
  "version": "2.0.19",
  "binary": "C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe",
  "displayName": "VdhCoApp",
  "description": "Video DownloadHelper companion app",
  "target": {
    "os": "windows",
    "arch": "x86_64",
    "node": "18"
  },
  "home": "C:\\Users\\Girard",
  "converterBinary": "C:\\Program Files\\DownloadHelper CoApp\\ffmpeg.exe",
  "converterBase": "ffmpeg",
  "converterBaseVersion": "c30f360955"
}

**Comment by GrampaWildWilly** [comment]
That's what you get when an item is protected by DRM.  You should search this forum for other discussions in which DRM is mentioned.

**Reply by GirardT** [reply]
It doesn't look like there is a workaround, is there?

**Reply by GrampaWildWilly** [reply]
Right.
